models/actions/runner.go

```diff
@@ -52,14 +52,50 @@ type ActionRunner struct {
 	Deleted timeutil.TimeStamp `xorm:"deleted"`
 }
 
-func (r *ActionRunner) OwnType() string {
+// RunnerType defines the runner type
+type RunnerType int //revive:disable-line:exported
```

Comment:
```suggestion
type RunnerType int //nolint:revive
```

Should use this shared style of disable comment.